### ISSUE TITLE
Use dataset name, rather than "file name" on organisation listing pages

### ISSUE THREAD

**Issue by andylolz** [IATI/validator-web#229]
**Is your feature request related to a problem? Please describe.**
For example, on the [OCHA Country Based Pooled Funds validator page](https://iativalidator.iatistandard.org/organisation/unocha-cbpf), the different datasets are indistinguishable from each other. That’s because the validator lists "file names", which are extracted from URLs.

The registry instead uses dataset name. See the [OCHA Country Based Pooled Funds registry page](https://iatiregistry.org/publisher/unocha-cbpf), where the different datasets have distinct names.

**Describe the solution you'd like**
Use dataset name, rather than "file name" on organisation listing pages.

**Describe alternatives you've considered**
Alternatively, use the registry dataset IDs. But names are supposed to be human-readable, so they’re preferable.

**Comment by simon-20** [comment]
@stevieflow , @robredpath - do we want to make this change on the Validator? We can roll it into the other updates to UI I'm doing. 

An alternative would be to display both Dataset short name and file name.

**Comment by stevieflow** [comment]
@simon-20 yes please

One thing I'm unclear of is whether a short name would exist in the cases, such as OCHA, where they register API endpoints, rather than specific *files* 

Thanks

**Comment by simon-20** [comment]
Yes, there is always a `short_name`, and there isn't any material difference between URL that points to a file or endpoint (we often call them files and often they are static files, but really it's just the response at the end of a web server).

**Comment by simon-20** [comment]
Resolved by commit b14d090adc51a42442aa08add5a259c6147bb94a which was released as part of 2.9.5
